homeassistant/components/lg_thinq/text.py

```diff
@@ -0,0 +1,112 @@
+"""Support for text entities."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+import logging
+
+from thinqconnect import DeviceType
+from thinqconnect.integration import ActiveMode, TimerProperty
+
+from homeassistant.components.text import TextEntity, TextEntityDescription
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import ThinqConfigEntry
+from .entity import ThinQEntity
+
+
+@dataclass(frozen=True, kw_only=True)
+class ThinQTextEntityDescription(TextEntityDescription):
+    """Describes ThinQ text entity."""
+
+    default_hint: str | None = None
+
+
+TIMER_ABSOLUTE_TO_START_TEXT_DESC = ThinQTextEntityDescription(
+    key=TimerProperty.ABSOLUTE_TO_START,
+    translation_key=TimerProperty.ABSOLUTE_TO_START,
+    default_hint="Input format 24-hour clock",
+)
+TIMER_ABSOLUTE_TO_STOP_TEXT_DESC = ThinQTextEntityDescription(
+    key=TimerProperty.ABSOLUTE_TO_STOP,
+    translation_key=TimerProperty.ABSOLUTE_TO_STOP,
+    default_hint="Input format 24-hour clock",
```

Comment:
So what do we put in here? What's possible?

Reply:
It is the absolute time (ex. 00:00 ~ 23:59)
The reason we didn't use 'time' entity is that we can't cancel with 'None'

Reply:
I am also wondering if `time` entities have you to deal with timezones. I think I'll discuss this with others since I don't think a text entity is a right way to display this (its too free form), you'll hear from me

Reply:
I am waiting for updates from you about this. :)

Reply:
Okay so I discussed this and we think that a text entity is a bad idea since its too free form and a time entity works better for this. But we have a question to make sure that that is the right choice:

Is it able to set only one of the two entities? Or do they both need to be set?

Reply:
It is able to set only one.
I have a question. 
Do we have a way to set None to time entity?
Or if there is a any new recommendation to cancel the timer, please let me know.

Reply:
@LG-ThinQ-Integration I think you can also use a `sensor` with `device_class: duration`, which supports units like `d`, `h`, `min`, and `s` for measurement, or alternatively, use `device_class: timestamp`.

Reply:
> @LG-ThinQ-Integration I think you can also use a `sensor` with `device_class: duration`, which supports units like `d`, `h`, `min`, and `s`, or alternatively, use `device_class: timestamp` (like [Home Connect](https://github.com/home-assistant/core/blob/7e2b72fa5e83ba58a00909b55ae1e92ad4721fb8/homeassistant/components/home_connect/sensor.py#L57-L68) do).

We use this to set the timer.
But we can't set with sensor which is read-only.

Reply:
> But we can't set with sensor which is read-only.

My bad. Then the `time` platform is probably the best option. I think that resetting the timer in this case can only be done with a separate `button`.

Reply:
@joostlek 
Do you agree with this idea?
Then we are going to add button entity and time entity.
In my guess, this would take some time and would not be included in the first release.
Please let me know your suggestion.

Reply:
> > But we can't set with sensor which is read-only.
> 
> My bad. Then the `time` platform is probably the best option. I think that resetting the timer in this case can only be done with a separate `button`.

btw, Thank you for the suggestion.